### PR TITLE
Increase max http request size

### PR DESCRIPTION
**Pull request by vasspilka**
Max http request from 1MB to 16MB (MEDIUMTEXT size)

Issue #51

### CODE REVIEW

**Comment by dionyziz** [comment]
Some end-to-end tests for this are easy to write and would be nice.

**Comment by vasspilka** [comment]
Ok, maybe I'll find time tommorow to do them. Studying for exams...

**Comment by vasspilka** [comment]
Tests done

File: backend/test.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    r = requests.get(config.BASE_URI)
+    return r.status_code
+
+
+def big_http_request():
+    global id
+    url = config.BASE_URI + '/api/automaton/create'
+    data = {'name': 'test', 'data': 10000 * '{"states":{"q_1":true}}'}
+    r = requests.post(url, data)
+    id = r.text
```
Comment (dionyziz) [inline backend/test.py]: Don't use globals, move to unit test class

**Comment by dionyziz** [comment]
Any progress with this?

**Comment by vasspilka** [comment]
Shouldn't be hard, I am just not sure what you want, posted you on FB 15 min ago

**Pull request merged**